### ISSUE TITLE
-no-ocamlfind should be used automatically

### ISSUE THREAD

**Issue by mshinwell** [camlp4/camlp4#14]
The camlp4 Makefile should not assume that ocamlfind is present.
I think the configure script should detect if ocamlfind is present, and if not, cause the Makefile to pass -no-ocamlfind to ocamlbuild.

**Comment by ghost** [comment]
It's already the case, in myocamlbuild.ml:

``` ocaml
let () =
  dispatch begin function
  | Before_options ->
    Options.use_ocamlfind := false

  | After_options ->
    Options.use_ocamlfind := false;
    Options.make_links := false
```

**Comment by ghost** [comment]
Fixed by 23319bd0859213eaf6b0048f1f1c296447b11432
